src/main/java/org/folio/rest/exception/FailedValidationException.java

```diff
@@ -0,0 +1,7 @@
+package org.folio.rest.exception;
+
+public class FailedValidationException extends RuntimeException{
```

Comment:
Isn't this name too broad? Or it'll be used for any kind of validation errors?

Reply:
yes, it's used for any kind of validation error e.g. "Invalid amount entered", "Requested amount exceeds remaining amount", "Amount must be positive"